Updated screenshot for Wagtail 3

KalobTaulien opened the pull request:
Updated the screenshot in the README to reflect Wagtail 3 changes. Also shows off the commenting feature. 

Used browserframe.com for the frame; Used TinyPNG to compress the image. 

I'll likely be making another version of this at a later date (probably animated) and this is to serve as a placeholder under other work is finalized.

lb- commented:
Looks good but I think we should merge this just before 3.0 release gets created maybe?
I added the 3.0 milestone.

KalobTaulien commented:
Updated the screenshot with the latest work. I used BrowserFrame and added a shadow like the original image and compressed the image. Should be ready for Wagtail 3's release now.
